### PR TITLE
Add support for specifying database name for postgres wallet

### PR DESCRIPTION
**Pull request by joevolleyball**
MultiWalletSingleTable Postgres wallet has default DB name 'Wallets'.  This allows the name to be configured during wallet creation, open and delete as well as when init'ing the storage type.

### CODE REVIEW

**Comment by jovfer** [comment]
@joevolleyball thank you for contribution! Please take a look on DCO check https://github.com/hyperledger/indy-sdk/pull/1979/checks?check_run_id=341364957. The commit history should be fixed a bit by you own to allow us to merge the PR

**Comment by jovfer** [comment]
@ianco please review

File: experimental/plugins/postgres_storage/src/postgres_storage.rs [hunk 1]
```diff
@@ -602,41 +613,90 @@ impl WalletStrategy for DatabasePerWalletStrategy {
...
         let url_base = PostgresStorageType::_admin_postgres_url(&config, &credentials);
-        let url = PostgresStorageType::_postgres_url(_WALLETS_DB, &config, &credentials);
-
-        let conn = postgres::Connection::connect(&url_base[..], postgres::TlsMode::None)?;
+        let url = PostgresStorageType::_postgres_url(wallet_db_name, &config, &credentials);
+        debug!("postgres_url: {:?}", url);
+        debug!("connecting to postgres, url_base: {:?}", url_base);
+        let conn = postgres::Connection::connect(&url_base[..], config.tls())?;
 
+        /*
```
Comment (ianco) [inline experimental/plugins/postgres_storage/src/postgres_storage.rs]: Please remove commented-out code

File: experimental/plugins/postgres_storage/src/postgres_storage.rs [hunk 2]
```diff
@@ -798,6 +868,9 @@ impl PostgresStorageType {
 
     fn _admin_postgres_url(config: &PostgresConfig, credentials: &PostgresCredentials) -> String {
         let mut url_base = "postgresql://".to_owned();
+        debug!("_admin_postgres_url, credentials.admin_account: {:?}", credentials.account);
```
Comment (ianco) [inline experimental/plugins/postgres_storage/src/postgres_storage.rs]: Please don't write admin credentials to the log file (especially the password)

**Comment by Patrik-Stas** [comment]
@ianco a bit off-topic to this pull request, but seeing this I would like to take opportunity to ask you about opinion on pgsql plugin pull request I am planning. I would like to create variation of  `MultiWalletSingleTable` strategy which would share connection pool across wallets. Currently if you try to open many postgress wallets (especially in short period of time), you run into issue that number of connections on pgsql is too high, causing it to drop further connections for a while. From my understanding this is because every wallet has its own connection pool. This doesn't seem to be ideal when opening many wallets - so I would like to share the r2d2 connection pool across all wallets. 
I was trying to implemented that but run into few Rust issues (i am still pretty beginner in Rust), so before I really commit into working on this, do you see any issue with such strategy?

**Comment by ianco** [comment]
@Patrik-Stas I don't see any issues with that approach, for the MultiWalletSingleTable scheme.  It's tricky sharing objects in Rust in shared libraries but the connection pool is supposed to support that ...

**Comment by ianco** [comment]
DCO is failing, remember to signoff your commits

File: experimental/plugins/postgres_storage/src/postgres_storage.rs [hunk 3]
```diff
@@ -480,6 +483,7 @@ struct MultiWalletMultiTableStrategy {}
 impl WalletStrategy for DatabasePerWalletStrategy {
     // initialize storage based on wallet storage strategy
     fn init_storage(&self, _config: &PostgresConfig, _credentials: &PostgresCredentials) -> Result<(), WalletStorageError> {
+        env_logger::try_init();
```
Comment (ianco) [inline experimental/plugins/postgres_storage/src/postgres_storage.rs]: warning: unused `std::result::Result` that must be used
   --> src/postgres_storage.rs:486:9
    |
486 |         env_logger::try_init();
    |         ^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: #[warn(unused_must_use)] on by default
    = note: this `Result` may be an `Err` variant, which should be handled

warning: unused `std::result::Result` that must be used
   --> src/postgres_storage.rs:652:9
    |
652 |         env_logger::try_init();
    |         ^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this `Result` may be an `Err` variant, which should be handled

**Comment by joevolleyball** [comment]
Closing pull request.  Re-forking and re-commiting my changes to take care of the sing-off issues (and collapse the many commits into one) and will create a new PR

**Pull request closed without merging**